packages/tempus/src/Tempus_StepperBDF2_decl.hpp

```diff
@@ -77,6 +78,9 @@ class StepperBDF2 : virtual public Tempus::StepperImplicit<Scalar>
     virtual void setObserver(
       Teuchos::RCP<StepperObserver<Scalar> > obs = Teuchos::null);
 
+    virtual Teuchos::RCP<StepperObserver<Scalar> > getObserver() const
+    { return this->stepperObserver_; }
+
```

Comment:
This is returning stepperObserver, a StepperObserverComposite, but the signature is SteperObserver?!  How is it compiling?

Reply:
Ok, StepperObserverComposite is a StepperObserver.